languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -102026,6 +102026,94 @@ The accident victim died from her injuries.
             <example correction="">This <marker>is</marker> right.</example>
             <example>This <marker>is</marker> conforming with my current knowledge.</example>
         </rule>
+        <rulegroup id="VERB_HERE_SINCE" name="perfect tense" default="temp_off">
```

Comment:
Have you tested this rule in the rule editor? There is a big pitfall with "since" being used in the sense of "because".

Tip: In that case, "since" is followed by a subject in form of a PRP, or a proper name…